javascript/src/main/components/Nav/AuthNav.js

```diff
@@ -1,11 +1,24 @@
 import React from "react";
 import { useAuth0 } from "@auth0/auth0-react";
+import { Navbar } from "react-bootstrap";
 import LoginButton from "./LoginButton";
 import LogoutButton from "./LogoutButton";
 
 const AuthNav = () => {
-  const { isAuthenticated } = useAuth0();
-  return <>{isAuthenticated ? <LogoutButton /> : <LoginButton />}</>;
+  const { user } = useAuth0();
+  if(user) {
+    const { name, picture, email } = user;
```

Comment:
This is a nice use of destructuring; I'd suggest not listing `email` if you don't plan on using it, as it causes the production build to fail.
```suggestion
    const { name, picture } = user;
```